cmd/tendermint/commands/run_node.go

```diff
@@ -103,7 +103,7 @@ func AddNodeFlags(cmd *cobra.Command) {
 func NewRunNodeCmd(nodeProvider nm.Provider) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "node",
-		Short: "Run the tendermint node",
+		Short: "Run the tendermint node\nneed to set `--mode validator` to run the node as validator",
```

Comment:
I don't think short description is supposed to have newline characters. Maybe use `Long:`?